typing_protocol_intersection/mypy_plugin.py

```diff
@@ -23,6 +23,19 @@
 class ProtocolIntersectionPlugin(mypy.plugin.Plugin):
     # pylint: disable=unused-argument
 
+    def report_config_data(
```

Comment:
Note to self: see https://mypy.readthedocs.io/en/stable/extending_mypy.html?highlight=report_config_data#current-list-of-plugin-hooks